.travis.yml

```diff
@@ -43,4 +43,4 @@ before_script:
   - drush runserver 127.0.0.1:8080 &
   - until netstat -an 2>/dev/null | grep '8080.*LISTEN'; do true; done
 
-script: drush test-run RESTful --uri=http://127.0.0.1:8080
+script: drush test-run RestfulCreateEntityTestCase --uri=http://127.0.0.1:8080
```

Comment:
Yeah the only way I can get the tests to run is on Travis and it was taking way to long to get to the failure.